lago/providers/libvirt/vm.py

```diff
@@ -238,14 +238,22 @@ def _shutdown(self, libvirt_cmd, ssh_cmd, msg):
                 LOGGER.debug('{} using libvirt'.format(msg))
                 libvirt_cmd(dom)
 
-    def defined(self):
-        flags = libvirt.VIR_CONNECT_LIST_DOMAINS_ACTIVE \
-            | libvirt.VIR_CONNECT_LIST_DOMAINS_TRANSIENT
+    def _get_domain(self, flags):
         dom_names = [
             dom.name() for dom in self.libvirt_con.listAllDomains(flags)
         ]
         return self._libvirt_name() in dom_names
 
+    def alive(self):
```

Comment:
alive == active. 
active in libvirt = online. But it may be indeed in non-running state.
I could change the wording, just decided it might be better to keep it consistent (there's also network alive function, though there's no such function in alive.